OpenRA.Mods.Common/Traits/BodyOrientation.cs

```diff
@@ -52,6 +52,10 @@ public WRot QuantizeOrientation(in WRot orientation, int facings)
 
 		public virtual WAngle QuantizeFacing(WAngle facing, int facings)
 		{
+            if (facings <= 0)
+            {
+                return facing;
+            }
```

Comment:
Seems to be working, tho formatting is a bit off. We don't use {}s in single line elements in this project. I think just copy pasting the check above in QuantizeOrientation, along with its comment would be better. So:
```
			// Quantization disabled
			if (facings == 0)
				return facing;
```

Reply:
completed